setup.cfg

```
@@ -40,6 +40,8 @@ install_requires =
     asgiref
     typing_extensions; python_version < "3.8"
     markupsafe>=1.1.1
+    pyarrow==6.0.1
+    databricks-sql-connector==2.0.2
```

```suggestion
    pyarrow>=6
    databricks-sql-connector
```

This is better if it “works” (in the sense that pip allows it… no idea if it would actually “work”).